osclib/common.py

```diff
@@ -6,7 +6,7 @@ def version_calculate():
     if path.exists('.git'):
```

Comment:
Needs to be relative to `path.dirname(__file__)` as well.

I'm wondering whether this works properly with symlinks as well because

```
~/.osc-plugins/osclib -> /home/fvogt/.osc-plugins/osclib -> /home/fvogt/osc-plugin-factory/osclib
```

Thus `~/.osc-plugins/osclib/common.py` might ends up trying to read from `~/.osc-plugins/.git`. Might need canonicalization.

Reply:
Should be done